client/src/pages/CreateSurvey/Questions.jsx

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  TextField,
+  Button,
+  Select,
+  InputLabel,
+  FormControl,
+  MenuItem,
+  Checkbox,
+} from '@material-ui/core';
+
+const Option = ({ optionIndex, questionIndex }) => {
+  const dispatch = useDispatch();
+  const { questions } = useSelector((state) => state.createSurveyReducer);
+
+  const { options } = questions[questionIndex];
+  const { text } = options[optionIndex];
+
+  const setOptionText = (event) => {
+    const payload = {};
```

Comment:
any preference for this syntax and not this one?

const payload = {
questionIndex,
optionIndex, 
text: event.target.value 
}
?

keys that have same name as their values can just be referenced once

Reply:
yeah literally no reason why i did it that way for these ones will change it